clarity/data/utils.py

```diff
@@ -71,7 +71,7 @@ def speechweighted_snr(target: np.ndarray, noise: np.ndarray) -> float:
     return sw_snr
 
 
-def sum_signals(signals: list) -> Union[np.ndarray, Literal[0]]:
+def sum_signals(signals: list) -> np.ndarray | Literal[0]:
```

Comment:
Neat, I like this syntax, more consistent with logical operators and saves having to import/use `Union`.